src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -442,7 +580,11 @@ def postprocess(self, model_outputs, decoder_kwargs: Optional[Dict] = None, retu
         else:
             skip_special_tokens = self.type != "ctc"
             text = self.tokenizer.decode(items, skip_special_tokens=skip_special_tokens)
-            if return_timestamps:
+            if return_timestamps and "Whisper" in self.model.__class__.__name__:
+                offsets = self.tokenizer.decode(items, skip_special_tokens=skip_special_tokens, output_offsets=True)[
```

Comment:
Why not call the offsets `chunks` directly? We could get rid of 
```
elif return_timestamps and "Whisper" in self.model.__class__.__name__:
    optional["chunks"] = offsets
```
this way.

Reply:
any update?

Reply:
Done.